README.md

```diff
@@ -112,6 +112,30 @@ _must_ detect whether they're present or not before using them. See [this issue]
 and [this wiki page](https://github.com/silverstripe-australia/silverstripe-queuedjobs/wiki/Defining-queued-jobs) for 
 more information
 
+## Memory limit configuration
+
+By default this task will run until it comes without 12MB of the memory limit configured by PHP, giving the application
+room to safely shutdown and restart.
+
+In some cases the buffer can be adjusted to be more robust (increased) or more performant (decreased). To adjust this
```

Comment:
It means it'll restart fewer times, although it'll come closer to the memory hard limit set by php.